From 1dd8f3631cc86fcf670f69befb5cd02e0b548da4 Mon Sep 17 00:00:00 2001
From: "plai@redhat.com" <plai@redhat.com>
Date: Mon, 23 Sep 2019 20:40:21 +0200
Subject: [PATCH 05/12] i386: Add CPUID bit and feature words for
 IA32_ARCH_CAPABILITIES MSR

RH-Author: plai@redhat.com
Message-id: <1569271227-28026-5-git-send-email-plai@redhat.com>
Patchwork-id: 90857
O-Subject: [RHEL7.8 qemu-kvm PATCH v6 04/10] i386: Add CPUID bit and feature words for IA32_ARCH_CAPABILITIES MSR
Bugzilla: 1709971
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

From: Robert Hoo <robert.hu@linux.intel.com>

Support of IA32_PRED_CMD MSR already be enumerated by same CPUID bit as
SPEC_CTRL.

At present, mark CPUID_7_0_EDX_ARCH_CAPABILITIES unmigratable, per Paolo's
comment.

Signed-off-by: Robert Hoo <robert.hu@linux.intel.com>
Message-Id: <1530781798-183214-3-git-send-email-robert.hu@linux.intel.com>
Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
(cherry picked from commit 3fc7c73139d2d38ae80c3b0bc963b1ac1555924c)
Signed-off-by: Paul Lai <plai@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 target-i386/cpu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target-i386/cpu.c b/target-i386/cpu.c
index ff0921f..838c616 100644
--- a/target-i386/cpu.c
+++ b/target-i386/cpu.c
@@ -174,7 +174,7 @@ static const char *cpuid_7_0_edx_feature_name[] = {
     NULL, NULL, NULL, NULL,
     NULL, NULL, NULL, NULL,
     NULL, NULL, "spec-ctrl", "stibp",
-    NULL, "arch-facilities", NULL, "ssbd",
+    NULL, "arch-facilities|arch-capabilities", NULL, "ssbd",
 };
 
 static const char *cpuid_80000008_ebx_feature_name[] = {
-- 
1.8.3.1

